### PR TITLE
More improvements to benchmark automation

### PR DESCRIPTION
**Pull request by JelteF**
This allows running multiple benchmarks very easily. You can start multiple
with a single command and you don't have to worry about them anymore after
starting them.

This also includes various other small improvements.

### CODE REVIEW

File: azure/README.md [hunk 1]
```diff
@@ -27,28 +49,31 @@ To clean up:
...
 `iaas-postgres.bicep` file (depending on the test that you are running).
+
+## Running benchmarks in bulk
+
+Often you want to compare multiple runs, and probably run the same benchmark a
+few times to get some statistically relevant results. This can be done easily by
+running the `bulk-run.sh` script.
+
+```bash
+./bulk-run.sh | tee -a results.csv
```
Comment (marcocitus) [inline azure/README.md]: This tool seems a bit odd given that it does a number of very specific set ups.

Comment (JelteF) [reply azure/README.md]: Changed it such that bulk-run.sh now takes a file containing the runs that you want to execute.

File: azure/README.md [hunk 2]
```diff
@@ -3,22 +3,44 @@
...
+
 ## Running benchmarks against Azure Database for PostgreSQL Hyperscale (Citus)
 
 To run a benchmark against Citus on the managed service, use the following
 command. The first argument is the name of the resource group that you want to
 use.
-```
-./run-benchmark.sh "$USER-hammerdb-hyperscale" hyperscale
+```bash
+./run.sh "$USER-hammerdb-hyperscale" hyperscale
```
Comment (marcocitus) [inline azure/README.md]: minor: I got a little confused when I tried this command in the root directory.

File: run.sh [hunk 3]
```diff
@@ -19,9 +19,9 @@ DEFAULT_CH_RUNTIME_IN_SECS=${DEFAULT_CH_RUNTIME_IN_SECS:-7200}
 source parse-arguments.sh
 mkdir -p results/
 
-psql -f sql/vacuum-ch.sql
-psql -f sql/vacuum-tpcc.sql
-psql -f sql/do-checkpoint.sql
+psql -P pager=off -f sql/vacuum-ch.sql
```
Comment (marcocitus) [inline run.sh:22]: I usually do psql -tAX to disable everything in scripts

Comment (JelteF) [reply run.sh:22]: afaict that's mostly useful when parsing the output, but that's not necassary here. It also still won't turn off the pager, which is what is important here.
```bash
psql -tAX -c 'select 1 from generate_series(0, 1000);'
```

**Review by marcocitus — APPROVED** [review]
These scripts are pretty rough in terms of UX, but they do seem to work very well.

File: azure/README.md (hunk 2 above)
Comment (marcocitus) [reply azure/README.md]: Could be nice to use a slightly different name

File: azure/README.md [hunk 4]
```diff
@@ -27,28 +49,41 @@ To clean up:
...
+
+## Running benchmarks in bulk
+
+Often you want to compare multiple runs, and probably run the same benchmark a
+few times to get some statistically relevant results. This can be done easily by
+running the `bulk-run.sh` script. It takes a file describing the configurations
+you want to run as input:
+
+```bash
+./bulk-run.sh runs.sample | tee -a results.csv
```
Comment (aamederen) [inline azure/README.md:79]: In addition to running multiple runs at once, is there a way to run a benchmark n times and reporting aggregate values like min, max, avg?

Comment (JelteF) [reply azure/README.md:79]: The first number on the line is how often you want to run that specific benchmark. So you can use that to get multiple lines for the same benchmark. Then you can paste the resulting csv in excel and get the min/max/avg using a PivotTable.

**Pull request merged**